src/tool_operate.c

```diff
@@ -1744,8 +1761,14 @@ static CURLcode operate_do(struct GlobalConfig *global,
                   strerror(errno));
         }
 
+        if(config->rootme) {
+          int rc = WEXITSTATUS(pclose(outs.stream));
```

Comment:
This is failing CI on macOS because `WEXITSTATUS` apparently expects its argument to be an lvalue (??). Also more relevantly, a) `WEXITSTATUS` isn't meaningfully defined if `WIFEXITED` isn't true: if it died from a signal you'll get some mangled version of the signal number and b) it's not usable as a curl error anyway. Try something like

    int wstatus = pclose(outs.stream);
    if (!result && (!WIFEXITED(wstatus) || WEXITSTATUS(wstatus) != 0)) {
        result = CURLE_WRITE_ERROR;
    }

which matches the cases below.